buildkite/bazelci.py

```diff
@@ -788,10 +788,16 @@ def trigger_new_build(self, commit, message=None, env={}):
         dict
             the metadata for the build
         """
+        pipeline_info = self.get_pipeline_info()
+        if not pipeline_info:
+            error = f"Cannot find pipeline info for pipeline {self.pipeline}."
+            self._log("SERIOUS", error)
+            raise BuildkiteException(error)
+
         url = self._NEW_BUILD_URL_TEMPLATE.format(self._org, self._pipeline)
         data = {
             "commit": commit,
-            "branch": "master",
+            "branch": pipeline_info["default_branch"],
```

Comment:
I'm not sure whether this value always exists. The Buildkite UI says "Leave blank if your pipeline has no default branch.", so it might be better to check and default to "master" in this case.

Reply:
I see, updated